frontend/src/metabase/embedding/embedding-hub/components/StepperWithCards/StepperWithCards.tsx

```diff
@@ -72,17 +87,21 @@ export const StepperWithCards = ({ steps }: { steps: StepperStep[] }) => {
                       ? card.clickAction.onClick
                       : undefined;
 
+                  const isNextCard = nextCard && nextCard.id === card.id;
+
                   return (
                     <CardAction card={card} key={card.id}>
                       <Card
                         className={cx(S.stepCard, {
                           [S.optionalStepCard]: card.optional,
                           [S.lockedStepCard]: card.locked,
+                          [S.nextStepCard]: isNextCard,
                         })}
                         component={onClick ? "button" : undefined}
                         onClick={onClick}
                         disabled={card.locked}
                         data-testid={`step-card-${card.id}`}
+                        data-next-step={isNextCard}
```

Comment:
Used for unit testing. I tried to use `[data-next-step]` in CSS but didn't work, so I used `.nextStepCard` for consistency.